releasenotes/notes/add-generate-preset-pm-global-import-efb12f185f3f738b.yaml

```
@@ -0,0 +1,11 @@
+---
+upgrade_transpiler:
```

I think this is probably an upgrade? It does increase usability. I am also not sure if "global import" is the best term, I also thought of "top level import".

---

It should probably a feature note since we're adding something new. An upgrade note would normally be reserved if we were changing something that might require user input when upgrading to the new version. This is just allowing a simpler import but the old path still works fine and is a supported part of our public api.

```suggestion
features_transpiler:
```